### PR TITLE
Add examples for bitwise logical operation functions

### PR DESCRIPTION
**Pull request by mkDoku**
- Add example for `bitwise_not`
- Add example for `bitwise_and`
- Add example for `bitwise_or`
- Add example for `bitwise_xor`
- Group the above functions according to their arity (unary/binary)

### CODE REVIEW

**Comment by mkDoku** [comment]
Add examples to be consistent in `bitwise.rst` ("Document functions for bitwise shifting #7654").

File: docs/src/main/sphinx/functions/bitwise.rst [hunk 1]
```diff
@@ -12,22 +12,42 @@ Bitwise functions
         SELECT bit_count(-7, 64); -- 62
         SELECT bit_count(-7, 8); -- 6
 
+.. function:: bitwise_not(x) -> bigint
```
Comment (mosabua) [inline docs/src/main/sphinx/functions/bitwise.rst:15]: Why are you reordering the functions to be not alphabetically sorted?

**Review by mosabua — CHANGES_REQUESTED** [review]
Looks good apart from the ordering.

File: docs/src/main/sphinx/functions/bitwise.rst (hunk 1 above)
Comment (mkDoku) [reply docs/src/main/sphinx/functions/bitwise.rst:15]: Sorry, my bad. I wasn't aware of the alphabetic ordering. My thinking was, that the logical operations (and, not, or and xor) could be sorted according to their arity (not - one argument, and - two arguments, or - two arguments, xor - two argument.)

I will revert this part.

**Comment by martint** [comment]
Thanks! Can you squash the commits?

**Comment by mkDoku** [comment]
Sorry, I am new to github. How would I do that (squashing the commits)?

**Comment by mkDoku** [comment]
> Thanks! Can you squash the commits?

Now it should be only 1 commit.

**Comment by mkDoku** [comment]
Sorry for the mess. This was the first time for me squashing commits and I forgot to change the commit message, twice.

**Review by mosabua — APPROVED** [review]
Looks great now.

**Pull request merged**